### ISSUE TITLE
HOG visualization

### ISSUE THREAD

**Issue by JHang2020** [mx-mark/VideoTransformer-pytorch#13]
How do you visualize hog feature? The output is a histogram, right?
Thanks!

**Comment by mx-mark** [comment]
@Eason-Great Yes, the prediction is the hog features. As for the hog visulization, we follow the source code in the scikit-image [https://github.com/scikit-image/scikit-image/blob/v0.19.0/skimage/feature/_hog.py#L48-L307].

**Comment by mx-mark** [comment]
@Eason-Great  Yes, the prediction is the hog features. As for the hog visulization, we follow the source code in the scikit-image https://github.com/scikit-image/scikit-image/blob/v0.19.0/skimage/feature/_hog.py#L48-L307
